apps/spruce/src/gql/generated/types.ts

```diff
@@ -234,14 +234,12 @@ export type CommitQueueParams = {
   __typename?: "CommitQueueParams";
   enabled?: Maybe<Scalars["Boolean"]["output"]>;
   mergeMethod: Scalars["String"]["output"];
-  mergeQueue: MergeQueue;
```

Comment:
This looks like it's unrelated?

Reply:
These are just updates to a generated file. (It will make the waterfall green again)